deploy/default-azuredeploy.json

```diff
@@ -34,17 +34,11 @@
     },
     "imageTag": {
       "type": "string",
-      "allowedValues": [
-        "v1.0.0",
-        "v2.0.0",
-        "v2.1.0"
-      ],
-      "defaultValue": "v2.1.0"
+      "defaultValue": "healthplatformregistry.azurecr.io/fhirconverter:v2.1.0"
```

Comment:
If we are storing the complete one here, maybe it does not belong to `imageTag`?

Reply:
Absolutely, the `imageTag` is not a good name,  I'll change it to `imagePath`.